Cannot open include file 'pcre2_internal.h' when building pcre2

I am getting an error when building pcl (`./vcpkg install pcl pcl[qt]`), the output shows:
```
Starting package 1/89: pcre2:x86-windows
Building package pcre2[core]:x86-windows...
-- CURRENT_INSTALLED_DIR=C:/Program Files (Portable)/vcpkg/installed/x86-windows
-- DOWNLOADS=C:/Program Files (Portable)/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=C:/Program Files (Portable)/vcpkg/packages/pcre2_x86-windows
-- CURRENT_BUILDTREES_DIR=C:/Program Files (Portable)/vcpkg/buildtrees/pcre2
-- CURRENT_PORT_DIR=C:/Program Files (Portable)/vcpkg/ports/pcre2/.
-- Using cached C:/Program Files (Portable)/vcpkg/downloads/pcre2-10.30.zip
-- Testing integrity of cached file...
-- Testing integrity of cached file... OK
-- Extracting done
-- Configuring x86-windows
-- Configuring x86-windows done
-- Build x86-windows-rel
CMake Error at C:/Program Files (Portable)/vcpkg/scripts/cmake/vcpkg_build_cmake.cmake:129 (message):
    Command failed: C:/Program Files/CMake/bin/cmake.exe;--build;.;--config;Release;--target;install;--;-v;-j1
    Working Directory: C:/Program Files (Portable)/vcpkg/buildtrees/pcre2/x86-windows-rel
    See logs for more information:
      C:\Program Files (Portable)\vcpkg\buildtrees\pcre2\install-x86-windows-rel-out.log

Call Stack (most recent call first):
  C:/Program Files (Portable)/vcpkg/scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  C:/Program Files (Portable)/vcpkg/ports/pcre2/portfile.cmake:23 (vcpkg_install_cmake)
  C:/Program Files (Portable)/vcpkg/scripts/ports.cmake:72 (include)


Error: Building package pcre2:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: pcre2:x86-windows
  Vcpkg version: 0.0.104-2018-02-07-d503a55412b3fa1b2b96ae0719286ee593f3d732

Additionally, attach any relevant sections from the log files above.
```
And when I looked into `install-x86-windows-rel-out.log`, I found 
```
[1/81] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1412~1.258\bin\Hostx64\x86\cl.exe   -DHAVE_CONFIG_H -DPCRE2_CODE_UNIT_WIDTH=8 -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -Dpcre2_8_EXPORTS -I. -I"C:\Program Files (Portable)\vcpkg\buildtrees\pcre2\src\pcre2-10.30" -I"C:\Program Files (Portable)\vcpkg\installed\x86-windows\include" /DWIN32 /D_WINDOWS /W3 /utf-8 /MP -IC:/Program Files (Portable)/vcpkg/buildtrees/pcre2/src/pcre2-10.30/src /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\pcre2-8.dir\pcre2_chartables.c.obj /FdCMakeFiles\pcre2-8.dir\ /FS -c pcre2_chartables.c
FAILED: CMakeFiles/pcre2-8.dir/pcre2_chartables.c.obj 
C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1412~1.258\bin\Hostx64\x86\cl.exe   -DHAVE_CONFIG_H -DPCRE2_CODE_UNIT_WIDTH=8 -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -Dpcre2_8_EXPORTS -I. -I"C:\Program Files (Portable)\vcpkg\buildtrees\pcre2\src\pcre2-10.30" -I"C:\Program Files (Portable)\vcpkg\installed\x86-windows\include" /DWIN32 /D_WINDOWS /W3 /utf-8 /MP -IC:/Program Files (Portable)/vcpkg/buildtrees/pcre2/src/pcre2-10.30/src /MD /O2 /Oi /Gy /DNDEBUG /Z7 /showIncludes /FoCMakeFiles\pcre2-8.dir\pcre2_chartables.c.obj /FdCMakeFiles\pcre2-8.dir\ /FS -c pcre2_chartables.c
Microsoft (R) C/C++ Optimizing Compiler Version 19.12.25835 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.

cl : Command line warning D9024 : unrecognized source file type 'Files', object file assumed
cl : Command line warning D9027 : source file 'Files' ignored
cl : Command line warning D9024 : unrecognized source file type '(Portable)/vcpkg/buildtrees/pcre2/src/pcre2-10.30/src', object file assumed
cl : Command line warning D9027 : source file '(Portable)/vcpkg/buildtrees/pcre2/src/pcre2-10.30/src' ignored
cl : Command line warning D9030 : '/showIncludes' is incompatible with multiprocessing; ignoring /MP switch
pcre2_chartables.c(27): fatal error C1083: Cannot open include file: 'pcre2_internal.h': No such file or directory
```

Any help?

--- a/ports/pcre2/CONTROL
+++ b/ports/pcre2/CONTROL
@@ -1,3 +1,3 @@
 Source: pcre2
-Version: 10.30-1
+Version: 10.30-2
 Description: PCRE2 is a re-working of the original Perl Compatible Regular Expressions library

--- a/ports/pcre2/fix-space.patch
+++ b/ports/pcre2/fix-space.patch
@@ -1,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index fbc37fe..54e25f2 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -91,7 +91,7 @@ CMAKE_POLICY(SET CMP0026 OLD)
+ 
+ SET(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake) # for FindReadline.cmake
+ 
+-SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -I${PROJECT_SOURCE_DIR}/src")
++SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -I\"${PROJECT_SOURCE_DIR}/src\"")
+ 
+ # external packages
+ FIND_PACKAGE( BZip2 )

--- a/ports/pcre2/portfile.cmake
+++ b/ports/pcre2/portfile.cmake
@@ -2,7 +2,7 @@ set(PCRE2_VERSION 10.30)
 include(vcpkg_common_functions)
 set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/pcre2-${PCRE2_VERSION})
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://sourceforge.net/projects/pcre/files/pcre2/${PCRE2_VERSION}/pcre2-${PCRE2_VERSION}.zip/download"
+    URLS "https://ftp.pcre.org/pub/pcre/pcre2-${PCRE2_VERSION}.zip" "https://sourceforge.net/projects/pcre/files/pcre2/${PCRE2_VERSION}/pcre2-${PCRE2_VERSION}.zip/download"
     FILENAME "pcre2-${PCRE2_VERSION}.zip"
     SHA512 03e570b946ac29498a114b27e715a0fcf25702bfc9623f9fc085ee8a3214ab3c303baccb9c0af55da6916e8ce40d931d97f1ee9628690563041a943f0aa2bc54)
 
@@ -19,6 +19,9 @@ vcpkg_configure_cmake(
         -DPCRE2_SUPPORT_UNICODE=ON
         -DPCRE2_BUILD_TESTS=OFF
         -DPCRE2_BUILD_PCRE2GREP=OFF)
+
+vcpkg_apply_patches(SOURCE_PATH ${SOURCE_PATH}
+    PATCHES ${CMAKE_CURRENT_LIST_DIR}/fix-space.patch)
 
 vcpkg_install_cmake()
 

